ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_security.schema.yml

```diff
@@ -60,6 +60,22 @@ keys:
             name:
               type: str
               description: Name of the SA policy. The "null" value is deprecated and will be removed in AVD 5.0.0
+            sa_lifetime:
+              type: dict
+              keys:
+                value:
+                  type: int
+                  description: |-
+                    Lifetime value for this SA.
+                    Valid range depends on the unit.
```

Comment:
Normally we list the valid ranges here. Search `cached_results_timeout` for one example.